README.md

```diff
@@ -190,6 +190,26 @@ if you use `volume` you must expose the volume in the `docker-compose.override.y
 
 The environment variables `DDE_DNS_FORWARD_1` and `DDE_DNS_FORWARD_2` allow for setting custom DNS servers in the Docker Development Environment (DDE). This is useful when local Internet DNS servers are to be used.
 
+### Configuring a Custom Shell
```

Comment:
@sbaerlocher as discussed,  this is currently in the middle of the dns documentation. Also the argument in the Dockerfile should be added to the documentation